### PR TITLE
Add missing word lists from BlackArch

### PR DESCRIPTION
**Pull request by zzJZzz**
### Details

- I created a ruby script to compare the repo.json file with the current wordlist.yml file and add anything that was missing. 
- I forgot to add the blank line in between so I had to create another script to do that. I have since modified the original script to add the new line.

### Testing
I also ran the rspec for the file and 5 green passing tests.

### CODE REVIEW

**Comment by postmodern** [comment]
@zzJZzz when you first edited the file it appears that your editor or something automatically changed the YAML schema of the entire file:

```
-  :url: https://github.com/urbanadventurer/WhatWeb/blob/master/plugin-development/alexa-top-1000.txt
-  :summary: The Alexa Top 1000 domain names.
-  :categories:
-    - dns
-    - domains
+  url: https://github.com/urbanadventurer/WhatWeb/blob/master/plugin-development/alexa-top-1000.txt
+  summary: The Alexa Top 1000 domain names.
+  categories:
+  - dns
+  - domains
 
...
```

The YAML schema must use Symbol keys (ex `:foo: ...`) to reduce memory consumption.

File: data/wordlists.yml [hunk 1]
```diff
@@ -8402,7 +8402,7 @@ Afrikaans_lower:
   categories:
   - misc
 
-[COMBO] Mail - Pass (Sql) 30858:
+Mail - Pass (Sql) 30858:
```
Comment (postmodern) [inline data/wordlists.yml]: Should probably include `Weakpas`. Probably not necessary to include the number of words or size of the wordlist in the wordlist name.

File: data/wordlists.yml [hunk 2]
```diff
@@ -9170,7 +9170,7 @@ DefUserNames:
   categories:
   - usernames
 
-01:
+one:
```
Comment (postmodern) [inline data/wordlists.yml]: This is too generic of a name. Should include "Weakpass" or something that indicates where the wordlist is from.

File: data/wordlists.yml [hunk 3]
```diff
@@ -10550,13 +10550,13 @@ uplay:
   categories:
   - misc
 
-13:
+thirteen:
```
Comment (postmodern) [inline data/wordlists.yml]: Too generic. Needs to indicate where the wordlist is from and what does it contain.

**Comment by zzJZzz** [comment]
Probably one of my code formatters..ok I can fix this. Apologies for missing it.

File: data/wordlists.yml [hunk 4]
```diff
@@ -10550,13 +10550,13 @@ uplay:
...
   url: https://download.weakpass.com/wordlists/768/13.dic.gz
   summary: Misc wordlist
   categories:
   - misc
 
-02:
+two:
```
Comment (postmodern) [inline data/wordlists.yml]: Too generic. Needs to indicate where the wordlist is from and what does it contain.

File: data/wordlists.yml [hunk 5]
```diff
@@ -5630,7 +5630,7 @@ pron:
   categories:
   - misc
 
-[COMBO] NEW:
+NEW:
```
Comment (postmodern) [inline data/wordlists.yml]: Too generic of a wordlist name. Should include either `Weakpass` or `COMBO` to indicate the specific wordlist.

File: data/wordlists.yml [hunk 6]
```diff
@@ -30614,7 +30614,7 @@ one-xl:
   categories:
   - fuzzing
 
-one:
+one01:
```
Comment (postmodern) [inline data/wordlists.yml:30617]: Too generic of a name. Should mention `SecLists` and `User-Agents`.

**Review by postmodern — CHANGES_REQUESTED** [review]

**Comment by postmodern** [comment]
I submitted some other suggestions. I think because the first commit accidentally changed the format of the `wordlists.yml` file, the PR should be redone and resubmitted.

**Pull request closed without merging**